editoast/src/views/rolling_stocks/mod.rs

```diff
@@ -124,27 +135,32 @@ pub struct RollingStockForm {
 impl From<RollingStockForm> for RollingStockModel {
```

Comment:
```suggestion
impl From<RollingStockForm> for Changeset<RollingStockModel> {
```

Makes more sense because the form does not provide all the fields, some are filled by the DB (such as the pk).

Check out the `flat_<field>` functions of changesets, they prove useful here.